src/main/java/com/minecolonies/coremod/colony/buildings/BuildingMiner.java

```diff
@@ -129,28 +129,28 @@
     /**
      * Max depth the miner reaches at level 3.
      */
-    private static final int                       MAX_DEPTH_LEVEL_3  = 5;
+    private static final int         MAX_DEPTH_LEVEL_3  = 5;
     /**
      * Stores the levels of the miners mine. This could be a map with (depth,level).
      */
     @NotNull
-    private final        List<Level>               levels             = new ArrayList<>();
+    private final        List<Level> levels             = new ArrayList<>();
     /**
      * True if shaft is at bottom limit.
      */
-    public               boolean                   clearedShaft       = false;
+    public               boolean     clearedShaft       = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make clearedShaft a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)